it/controllers/correctReturn/PackAtBusinessAddressControllerISpec.scala

```diff
@@ -41,6 +46,186 @@ class PackAtBusinessAddressControllerISpec extends ControllerITTestHelper {
+          disableTranslations = Some(true),
+          allowedCountryCodes = None,
+          confirmPageConfig = Some(ConfirmPageConfig(
+            showSearchAgainLink = Some(true),
+            showSubHeadingAndInfo = Some(true),
+            showChangeLink = Some(true),
+            showConfirmChangeText = Some(true)
+          )),
+          timeoutConfig = Some(TimeoutConfig(
+            timeoutAmount = 900,
+            timeoutUrl = controllers.auth.routes.AuthController.signOut.url,
+            timeoutKeepAliveUrl = Some(controllers.routes.KeepAliveController.keepAlive.url)
+          )),
+          serviceHref = Some(controllers.routes.IndexController.onPageLoad.url),
+          pageHeadingStyle = Some("govuk-heading-l")
+        ),
+        labels = Some(
+          JourneyLabels(
+            en = Some(LanguageLabels(
+              appLevelLabels = Some(AppLevelLabels(
+                navTitle = Some("Soft Drinks Industry Levy"),
+                phaseBannerHtml = None
+              )),
+              selectPageLabels = None,
+              lookupPageLabels = Some(
+                LookupPageLabels(
+                  title = Some("Find UK warehouse address"),
+                  heading = Some("Find UK warehouse address"),
+                  postcodeLabel = Some("Postcode"))),
+              editPageLabels = Some(
+                EditPageLabels(
+                  title = Some("Enter the UK warehouse address"),
+                  heading = Some("Enter the UK warehouse address"),
+                  line1Label = Some("Address line 1"),
+                  line2Label = Some("Address line 2"),
+                  line3Label = Some("Address line 3 (optional)"),
+                  townLabel = Some("Address line 4 (optional)"),
+                  postcodeLabel = Some("Postcode"),
+                  organisationLabel = Some("Trading name (optional)"))
+              ),
+              confirmPageLabels = None,
+              countryPickerLabels = None
+            ))
+          )),
+        requestedVersion = None
+      )
+      val expectedResultInDB: Some[JsObject] = Some(
+        Json.obj("correctReturn" -> Json.obj( "packAtBusinessAddress" -> false)
+        ))
+
+      val alfOnRampURL: String = "http://onramp.com"
+
+      given
+        .commonPrecondition
+        .alf.getSuccessResponseFromALFInit(alfOnRampURL)
+      setAnswers(emptyUserAnswersForCorrectReturn)
+
+      WsTestClient.withClient { client =>
+        val result = createClientRequestPOST(
+          client, correctReturnBaseUrl + normalRoutePath, Json.obj("value" -> "false")
+        )
+
+        whenReady(result) { res =>
+          res.status mustBe 303
+          res.header(HeaderNames.LOCATION) mustBe Some(alfOnRampURL)
+          getAnswers(sdilNumber).map(userAnswers => userAnswers.data) mustBe expectedResultInDB
+          ALFTestHelper.requestedBodyMatchesExpected(wireMockServer, journeyConfigToBePosted) mustBe false
```

Comment:
These will need to be true - there is something needs to be updating in the journeyConfig to make this so.